OpenSim/Actuators/VandenBogert2011Muscle.cpp

```diff
@@ -0,0 +1,1234 @@
+/* -------------------------------------------------------------------------- *
+ *                      OpenSim:  VandenBogert.cpp                            *
+ * -------------------------------------------------------------------------- */
+
+
+//=============================================================================
+// INCLUDES
+//=============================================================================
+#include "VandenBogert2011Muscle.h"
+#include <OpenSim/Simulation/Model/Model.h>
+
+
+//=============================================================================
+// STATICS
+//=============================================================================
+using namespace std;
+using namespace OpenSim;
+
+
+
+//=============================================================================
+// CONSTRUCTOR(S) AND DESTRUCTOR
+//=============================================================================
+//_____________________________________________________________________________
+/*
+ * Default constructor
+ */
+VandenBogert2011Muscle::VandenBogert2011Muscle()
+{
+    constructProperties();
+}
+
+//_____________________________________________________________________________
+/*
+ * Constructor.
+ */
+
+VandenBogert2011Muscle::VandenBogert2011Muscle(const std::string& aName)
```

Comment:
C++11 has "delegating constructors": you can call another constructor from a constructor to reduce duplication:

```cpp
VandenBogert2011Muscle(const std::string& name) : VandenBogert2011Muscle() {
    setName(name);
}
```